Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -168,6 +171,20 @@ extension _EasyHandle {
         let protocols = (CFURLSessionProtocolHTTP | CFURLSessionProtocolHTTPS)
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPROTOCOLS, protocols).asError()
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionREDIR_PROTOCOLS, protocols).asError()
+        #if os(Android)
+            // See https://curl.haxx.se/docs/sslcerts.html
+            // For SSL to work you need "cacert.pem" to be accessable
+            // at the path pointed to by the URLSessionCAInfo env var.
```

Comment:
How does this look. I’ve left the old api in as I prefer it and used it in the new api. Not entirely happy that setting sslCertificateAuthorityFile to nil turns off SSL PEER validaton so easilly also should it be sslCertificateAuthorityURL or sslCertificateAuthorityInfo?

```swift
#if os(Android)
extension URLSession {
    fileprivate static var _sslCertificateAuthorityFile: UnsafeMutablePointer<Int8>?
    public static func setCertificateAuthorityInfo(filePath: String) {
        free(URLSession._sslCertificateAuthorityFile)
        filePath.withCString {
            URLSession._sslCertificateAuthorityFile = strdup($0)
        }
    }
    public static var sslCertificateAuthorityFile: URL? {
        set(value) {
            guard let value = value else {
                setCertificateAuthorityInfo(filePath: "UNSAFE_SSL_NOVERIFY")
                NSLog("*** SSL PEER verfication disabled ***")
                return
            }
            guard value.isFileURL && FileManager.default.fileExists(atPath: value.path) else {
                NSLog("*** sslCertificateAuthorityFile not FileURL or does not exist ***")
                return
            }
            setCertificateAuthorityInfo(filePath: value.path)
        }
        get {
            return URL(fileURLWithPath: String(cString: URLSession._sslCertificateAuthorityFile!))
        }
    }
}
#endif
```